INSTALL.md

```diff
@@ -54,189 +54,198 @@ to stop the application, run:
 ```
-
-Run [Bundler](https://bundler.io/) to install all required Ruby gems:
-
-```shell
-bundle install
+gem install bundler
 ```
 
-If the `mysql2` gem installation fails with the error:
-
-```
-ld: library not found for -lssl
+Then configure the bundle config with:
+```sh
+bundle config --global build.eventmachine --with-cppflags="-I$(brew --prefix openssl)/include"
+bundle config --global build.mysql2 --with-opt-dir="$(brew --prefix openssl)"
+bundle config --local build.github-markdown --with-cflags=-Wno-error=implicit-function-declaration
+bundle config --local build.thin --with-cflags=-Wno-error=implicit-function-declaration
 ```
 
-you can fix it setting the flags:
-
-```shell
-bundle config --local build.mysql2 "--with-ldflags=-L/usr/local/opt/openssl/lib --with-cppflags=-I/usr/local/opt/openssl/include"
+And finally install all gems:
+```
+bundle install
 ```
 
-and run `bundle install` again.
+> If the `mysql2` gem installation fails with the error:
+>
+> ```
+> ld: library not found for -lssl
+> ```
+>
+> you can fix it setting the flags:
+>
+> ```sh
+> bundle config --local build.mysql2 --with-ldflags="-L$(brew --prefix openssl)/lib" --with-cppflags="-I$(brew --prefix openssl)/include"
+> ```
+>
+> and run `bundle install` again.
 
-#### Node packages
+#### Yarn
 
-Install [Yarn](https://yarnpkg.com/):
+Install [Yarn](https://yarnpkg.com/) to manage all required Javscript packages:
 
-```bash
+```
 brew install yarn
 ```
 
+And install them:
 
-Run [Yarn](https://www.yarnpkg.com/) to install all the required dependencies:
-
-```bash
+```
 yarn install
 ```
 
 ## Manual setup on Fedora (34)
 
 ### Prerequisites
 
-#### Ruby version
-
-The project supports **2.6.x**.
+#### Ruby and Node.js
 
-Verify you have a proper version by running on your terminal:
+The project supports **[ruby 2.6.x](https://www.ruby-lang.org/en/downloads/)** and **[Node.js 12](https://nodejs.org/en/download/)**. Verify you have a proper version by running on your terminal:
 
-```bash
-ruby -v
+```
+ruby -v && node -v
 ```
 
-> You can use [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install ruby.
+[asdf](https://asdf-vm.com/guide/getting-started.html#global) is a convenient tool version manager that reads
+from the `.tool-versions` file included in this project. This way you won't have to worry about the versioning
+of these packages and it will override the system's ruby version. After installing the required dependencies
+and asdf you need to install a plugin for each individual package:
 
-#### Node version
+```
+apt install curl git
+git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.10.2
 
-The project supports **Version: 12.X.X**.
+asdf plugin add ruby https://github.com/asdf-vm/asdf-ruby.git
+asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+```
 
-```bash
-dnf module install nodejs:12
+Finally, download the specific versions and use them in the local folder:
 ```
+asdf install
```

Comment:
Missing `ln -s .tool-versions.sample .tool-versions` step?

Reply:
Actually the whole section is still using the old format, I will update it.